packages/common/src/components/Footer/LinkGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from './Link';
+import media from '../../utils/media';
+
+const Column = styled.div`
```

Comment:
These components should go in a `elements.ts` file next to this file.
You can create a folder and make this file the `index.tsx`.